core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -105,7 +105,7 @@
  */
 
 #define DRAM0_BASE		0x00000000
-#define DRAM0_SIZE		0x3F000000
+#define DRAM0_SIZE		CFG_TZDRAM_START
```

Comment:
size = start? This doesn't seem to make much sense does it?

Reply:
You're right, sorry I did not see that. Would you mind adding a comment `/* DRAM0 exists below TZDRAM */`?

Reply:
For consistency and for future new readers, prefer: `#define DRAM0_SIZE (CFG_TZDRAM_START - DRAM0_BASE)` even if the result is the same in the generated image.